platform/iphone/gl_view.mm

```diff
@@ -263,57 +88,60 @@ + (Class)layerClass {
 
 //The GL view is stored in the nib file. When it's unarchived it's sent -initWithCoder:
 - (id)initWithCoder:(NSCoder *)coder {
-	active = FALSE;
-	if ((self = [super initWithCoder:coder])) {
-		self = [self initGLES];
+	if (self = [super initWithCoder:coder]) {
+		[self setUp];
 	}
 	return self;
 }
 
-- (id)initGLES {
+- (id)initWithFrame:(CGRect)frame {
+	if (self = [super initWithFrame:frame]) {
+		[self setUp];
+	}
+	return self;
+}
+
+- (void)setUp {
+	_instance = self;
+
+	self.useCADisplayLink = bool(GLOBAL_DEF("display.iOS/use_cadisplaylink", true)) ? YES : NO;
```

Comment:
Again, I would love to drop this `BOOL` and the `NSTimer` and adopt a `CADisplayLink` only strategy.